qmat/qcoeff/__init__.py

```diff
@@ -116,18 +136,18 @@ def register(cls:QGenerator)->QGenerator:
             cls(**params)
         except:
             raise TypeError(
-                f"{cls.__name__} could not be instanciated with DEFAULT_PARAMS")
+                f"{cls.__name__} could not be instantiated with DEFAULT_PARAMS")
     # Store class (and aliases)
     storeClass(cls, Q_GENERATORS)
     return cls
 
-def genQCoeffs(qType, withS=False, hCoeffs=False, **params):
+def genQCoeffs(qType, withS=False, hCoeffs=False, embedded=False, **params):
     try:
         Generator = Q_GENERATORS[qType]
     except KeyError:
-        raise ValueError(f"qType={qType} is not available")
+        raise ValueError(f"{qType=!r} is not available")
```

Comment:
Ah, that's where the notebook error comes from !

Reply:
Why did you do that ?

Reply:
Weird, it works locally. It's just more fancy string formatting. the `=` will print `<name of variable>=<str(variable)>`. So `f"{qtype=}"` is the same as `f"qtype={qtype}"`. The `!r` prints some quotes around it to emphasise that it's a string.

Reply:
It's due to a jupyter and notebook version apparently, that don't use the quote with `!r`. I fixed it with the last commit on main